src/modules/visits/services/queries.gql.ts

```diff
@@ -352,16 +480,35 @@ export const FIND_APPROVED_ENDED_VISITS_WITHOUT_NOTIFICATION = `
 				full_name
 			}
 			space {
+				id
+				schema_image
+				schema_color
+				schema_audience_type
+				schema_description
+				schema_public_access
+				schema_service_description
+				is_published
+				published_at
+				created_at
+				updated_at
```

Comment:
You're fetching more than you're using in the adapt function.
Is that with an eye on future features? or should we fetch less data?

Reply:
Consistency > needless optimisation